llama_stack/apis/inference/inference.py

```diff
@@ -239,7 +247,9 @@ async def chat_completion(
         response_format: Optional[ResponseFormat] = None,
         stream: Optional[bool] = False,
         logprobs: Optional[LogProbConfig] = None,
-    ) -> Union[ChatCompletionResponse, ChatCompletionResponseStreamChunk]: ...
+    ) -> Union[
```

Comment:
FINALLY! we actually type-hint it in the way it is supposed to be. 

This might become an issue with our OpenAPI generator, but we will fix that downstream. Our source must be always correct.